static/js/around-map.js

```diff
@@ -10,8 +16,17 @@ map.addControl(new mapboxgl.NavigationControl())
 map.addControl(new mapboxgl.GeolocateControl())
 
 // detectionLocation
-if(currentLocation.length == 0) {
-  focusUser()
+if (currentLocation.length == 0) {
+    navigator.geolocation.getCurrentPosition(position => {
+      localStorage.setItem('userPosition', [position.coords.latitude, position.coords.longitude])
+      focusUser()
+    }, function errorCallback(error) {
+      localStorage.setItem('errorPosition', error.code)
```

Comment:
See the comment at https://github.com/neomad-team/neomad.org/pull/88/commits/454fed60294f41a558148d2b9bc84a6a7d90c568#diff-254e6874b9742592d78e6cb2c1dfc9caR3